configuration/authentication/oauth2.md

```diff
@@ -210,3 +210,26 @@ auth:
         custom-params:
           type: oauth
 ```
+
+### GoAuthentic
```

Comment:
isn't it just 'authentic' rather than 'goauthentic'? I belive the use it as an org name rather then product name

Reply:
Yes, that’s correct. The proper name is Authentic (at least the developer is trying to achieve that), but I decided to use GoAuthentic to simplify search and user understanding, as search engines still struggle to provide the correct answer when using Authentic. Use any name you find appropriate. Thank you.